packages/nextjs-mf/utils/index.ts

```diff
@@ -1,6 +1,6 @@
 export { extractUrlAndGlobal, injectScript } from '@module-federation/utilities';
 export const revalidate = () => {
-  if(typeof window !== 'undefined') {
+  if(window && typeof window !== 'undefined') {
```

Comment:
Typeof should be good enough. Why add check for window in the first place. Typeof would be undefined causing falsy